apm-agent-core/src/main/java/co/elastic/apm/agent/configuration/CoreConfiguration.java

```diff
@@ -261,7 +258,7 @@
     private final ConfigurationOption<List<MethodMatcher>> traceMethods = ConfigurationOption
         .builder(new ListValueConverter<>(MethodMatcherValueConverter.INSTANCE), List.class)
         .key("trace_methods")
-        .tags("added[1.3.0,Enhancements in 1.4.0]")
+        .tags("added[1.3.0,Enhancements in 1.4.0 and 1.6.0]")
```

Comment:
`1.7.0` now
But I'm not sure if that can be considered an enhancement to the `trace_methods` directly.

Reply:
> But I'm not sure if that can be considered an enhancement to the `trace_methods` directly.

Why is that? It is only related to `trace_methods` and has no effect when not used. It extends the configuration capabilities of this feature.